subprojects/docs/src/docs/userguide/ideaPlugin.adoc

```diff
@@ -58,97 +56,66 @@ The IDEA plugin also adds a `cleanIdea` task to the project. This task deletes t
 The IDEA plugin adds the tasks shown below to a project. Notice that the `clean` task does not depend on the `cleanIdeaWorkspace` task. This is because the workspace typically contains a lot of user specific temporary data and it is not desirable to manipulate it outside IDEA.
 
 [[ideatasks]]
-.IDEA plugin - Tasks
-[cols="a,a,a,a", options="header"]
-|===
-| Task name
-| Depends on
-| Type
-| Description
-
-| `idea`
-| `ideaProject`, `ideaModule`, `ideaWorkspace`
-| -
-| Generates all IDEA configuration files
-
-| `openIdea`
-| `idea`
-| -
-| Generates all IDEA configuration files and opens the project in IDEA
-
-| `cleanIdea`
-| `cleanIdeaProject`, `cleanIdeaModule`
-| api:org.gradle.api.tasks.Delete[]
-| Removes all IDEA configuration files
-
-| `cleanIdeaProject`
-| -
-| api:org.gradle.api.tasks.Delete[]
-| Removes the IDEA project file
-
-| `cleanIdeaModule`
-| -
-| api:org.gradle.api.tasks.Delete[]
-| Removes the IDEA module file
-
-| `cleanIdeaWorkspace`
-| -
-| api:org.gradle.api.tasks.Delete[]
-| Removes the IDEA workspace file
-
-| `ideaProject`
-| -
-| api:org.gradle.plugins.ide.idea.GenerateIdeaProject[]
-| Generates the `.ipr` file. This task is only added to the root project.
-
-| `ideaModule`
-| -
-| api:org.gradle.plugins.ide.idea.GenerateIdeaModule[]
-| Generates the `.iml` file
-
-| `ideaWorkspace`
-| -
-| api:org.gradle.plugins.ide.idea.GenerateIdeaWorkspace[]
-| Generates the `.iws` file. This task is only added to the root project.
-|===
+`idea` — depends on: `ideaProject`, `ideaModule`, `ideaWorkspace`::
```

Comment:
:o: This "depends on" formatting is slightly different than what we have for the [Java Plugin doc](https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_tasks). Consistency would be nice to have but not critical.

Reply:
It's consistent with what I did for the Base Plugin chapter :wink: I agree with consistency. My issue with the other format is that it's just a bit too cramped and I want the task name to stand out with some space around it. Otherwise, I like how it mimics the Groovy syntax for declaring a new typed task. Maybe a compromise of:

  *classes* — (type: Task, dependsOn: compileJava, processResources)

Although in this case, it looks like normal syntax, but I think it's invalid (need square brackets around the other task names and no one specifies a task type of `Task`). If it's not exactly the same as the syntax in the build script, then I'd prefer to keep it distinct to avoid confusion and an potential for copy & paste errors.

Thoughts?